src/js/containers/search/filters/KeywordContainer.jsx

```diff
@@ -0,0 +1,70 @@
+/**
+  * KeywordContainer.jsx
+  * Created by Emily Gullo 11/30/16
+  **/
+
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as searchFilterActions from 'redux/actions/search/searchFilterActions';
+
+import Keyword from 'components/search/filters/keyword/Keyword';
+
+const propTypes = {
+    updateTextSearchInput: React.PropTypes.func,
+    clearFilterType: React.PropTypes.func,
+    keyword: React.PropTypes.string
+};
+
+export class KeywordContainer extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            newKeyword: '',
+            keyword: ''
+        };
+
+        this.submitText = this.submitText.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(e) {
```

Comment:
We shouldn't need two methods to grab text from the user input. I'd add a `ref` on the text input in `Keyword`, as well as a function that grabs the input from the `ref` when the submit button is clicked. That function will call `this.props.submitText` with the user-input value from the `ref` input, which will remove the need for both `handleChange` and keeping track of input in `state`.